src/components/common/RadioButtons.vue

```diff
@@ -21,21 +21,14 @@ export default {
   components: {},
   props: {
     icons: {
-      type: Array,
+      type: [Object, Array],
```

Comment:
unikaj mieszania typów. myślę, że możesz się zdecydować czy to będzie array czy obiekt. nawet jeśli tylko jeden element miałby być, to przyjedzie w arrayu z jednym elementem